src/test/java/emissary/server/EmissaryServerIT.java

```diff
@@ -42,7 +41,7 @@ public void testSSLWorks() throws Exception {
             String endpoint = cmd.getScheme() + "://" + hostPort + "/api/version";
             MapResponseEntity versionMap = client.send(new HttpGet(endpoint)).getContent(MapResponseEntity.class);
             Map<String, String> response = versionMap.getResponse();
-            assertThat(response.get(hostPort), equalTo(new Version().getVersion()));
+            assertEquals(new Version().getVersion(), response.get(hostPort));
         } catch (Exception e) {
```

Comment:
Optional cleanup:   discard the `catch` clause, which could be masking test failures